{"name": "test_almightycode[caseinfo0]", "status": "broken", "statusDetails": {"message": "NameError: name 'params' is not defined", "trace": "self = <test_case.login_test.Testwork object at 0x10f44f460>\ncaseinfo = {'name': '万能验证码', 'request': {'headers': {'Content-Type': 'application/json', 'User-Agent': 'Mozilla/5.0 (Windows NT 1...840.99 Safari/537.36', 'deviceId': '1254'}, 'method': 'get', 'params': 'none', 'path': '/api/admin/get/almighty/code'}}\n\n    @allure.story('验证码接口')\n    @pytest.mark.parametrize(\"caseinfo\", YamlUtil().yaml_read(\"data\", \"almightycode.yaml\"))\n    def test_almightycode(self, caseinfo):\n        # 获取ymal\n        value = assignment_yamlparams(caseinfo['request'])\n        url = URL + value['path']\n>       response = sendRequest(value['method'], url, value['params'], value['headers'])\n\ntest_case/login_test.py:27: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntools/base_requests.py:45: in sendRequest\n    request_log(method,url,headers,data)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nmethod = 'get', url = 'http://api.errand.u57.net/api/admin/get/almighty/code'\ndata = {'Content-Type': 'application/json', 'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (HTML, like Gecko) \"\"Chrome/54.0.2840.99 Safari/537.36', 'deviceId': '1254'}\njson = 'none', headers = None, files = None, cookies = None, kwargs = {}\n\n    def request_log( method, url,data=None, json=None,  headers=None, files=None, cookies=None,\n                        **kwargs):\n        logger.info(\"接口请求地址 ==>> {}\".format(url))\n        logger.info(\"接口请求方式 ==>> {}\".format(method))\n        # Python3中，json在做dumps操作时，会将中文转换成unicode编码，因此设置 ensure_ascii=False\n        logger.info(\"接口请求头 ==>> {}\".format(complexjson.dumps(headers, indent=4, ensure_ascii=False)))\n>       logger.info(\"接口请求 params 参数 ==>> {}\".format(complexjson.dumps(params, indent=4, ensure_ascii=False)))\nE       NameError: name 'params' is not defined\n\ntools/base_requests.py:78: NameError"}, "attachments": [{"name": "log", "source": "31dbd898-d953-4bbb-9604-f454192c16c3-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "caseinfo", "value": "{'name': '万能验证码', 'request': {'method': 'get', 'path': '/api/admin/get/almighty/code', 'headers': {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (HTML, like Gecko) \"\"Chrome/54.0.2840.99 Safari/537.36', 'deviceId': '1254', 'Content-Type': 'application/json'}, 'params': 'none'}}"}], "start": 1651808816259, "stop": 1651808816259, "uuid": "d2ef44a0-dde5-44c8-81a6-45f8bf26942b", "historyId": "6e7af39bb606afac8bf5087e4fe0bc35", "testCaseId": "f0f21a59f56785ca69b364ca5a72b296", "fullName": "test_case.login_test.Testwork#test_almightycode", "labels": [{"name": "story", "value": "验证码接口"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "login_test"}, {"name": "subSuite", "value": "Testwork"}, {"name": "host", "value": "xudeMac-mini.local"}, {"name": "thread", "value": "15796-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.login_test"}]}